custom_metrics_extractor/README.md

```diff
@@ -0,0 +1,29 @@
+# Custom metrics extractor
+This script generates a file containing all the custom metrics for an org. 
+
+It is related to [an older script to generate custom metrics](https://github.com/DataDog/se-docs/wiki/Pulling-the-list-of-custom-metrics----SCRIPT) with the added functionality to filter out core integration metrics and provides a count of total custom metrics.
```

Comment:
This is an internal only reference, but this repo is public -- so maybe no need to expand on it as to not confuse customers -- can put this reference in our standup card though.